p2p/conn/connection.go

```diff
@@ -319,6 +341,12 @@ FOR_LOOP:
 			legacy.WriteOctet(packetTypePing, c.bufWriter, &n, &err)
 			c.sendMonitor.Update(int(n))
 			c.flush()
+			c.Logger.Debug("Starting pong timer")
+			c.pongTimer.Reset(c.config.PongTimeout)
+		case <-c.pongTimer.C:
+			c.Logger.Debug("Pong timeout")
+			// XXX: should we decrease peer score instead of closing connection?
```

Comment:
?

Reply:
no scoring for now, lets just close the conn

Reply:
lets just remove the comment